test/Microsoft.ML.CodeAnalyzer.Tests/Helpers/AdditionalMetadataReferences.cs

```diff
@@ -16,10 +16,10 @@ internal static class AdditionalMetadataReferences
     {
 #if NETCOREAPP
         internal static readonly ReferenceAssemblies DefaultReferenceAssemblies = ReferenceAssemblies.Default
-            .AddPackages(ImmutableArray.Create(new PackageIdentity("System.Memory", "4.5.1")));
+            .AddPackages(ImmutableArray.Create(new PackageIdentity("System.Memory", "4.6.0")));
```

Comment:
What's this for?  Might not work correctly since that package doesn't exist yet.

Reply:
It looked to me like the roslyn test framework is trying to download these on the test machine.  So it would need a nuget source that has the packages.  I think we could probably make it work by copying the nuget.config.  Or you could just leave this change out.